Eatery/Onboarding/Controllers/OnboardingInfoViewController.swift

```diff
@@ -37,6 +38,14 @@ class OnboardingInfoViewController: OnboardingViewController {
         }
     }
 
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        DispatchQueue.main.async {
```

Comment:
Why delay the animation play?

Reply:
Ran into a bug where the animation would not play when the next viewController was presented in the pageViewController if we do not do it this way.